modules/vm/variables.tf

```diff
@@ -71,7 +90,6 @@ variable "os_disk" {
 
 variable "infrastructure_admin_username" {
   type = string
-  default = "azureuser"
```

Comment:
curious why this default value was removed?

Reply:
creating var for infrastructure_admin_username in root module https://github.com/ansible-content-lab/azure_terraform_deployment/pull/10/files#diff-05b5a57c136b6ff596500bcbfdcff145ef6cddea2a0e86d184d9daa9a65a288eR22-R27
and passing it https://github.com/ansible-content-lab/azure_terraform_deployment/pull/10/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR94 on each vm module, so no need to set default inside module

Reply:
understood, thanks